examples/misc_exporter_gcode.html

```diff
@@ -0,0 +1,264 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - exporter - Polyslice</title>
```

Comment:
Please add the new example to `files.json`.

https://github.com/mrdoob/three.js/blob/dev/examples/files.json

We also need a E2E screenshot. You can generate it via:
```
npm run make-screenshot misc_exporter_gcode
```

Reply:
Done in commit 3faab9b